content/aips/SUPPLY-BORROW-CAPS-UPDATE-AAVE-V3-02-12-2023.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Chaos Labs Supply and Borrow Cap Updates - Aave V3 Polygon and Arbitrum - 2023.02.07
```

Comment:
would recommend keeping this short & on point as it will be used on uis as title (also date might not match tz wise for ppl checking etc)
```suggestion
title: Aave V3 Polygon and Arbitrum
```

Reply:
updated to "Supply/Borrow Cap Updates Polygon and Arbitrum V3"